### PR TITLE
Application

### PR DESCRIPTION
**Pull request by hswiergiel**

### CODE REVIEW

**Review by FlorianBartier — DISMISSED** [review]
Review OK

File: frontend/src/pages/MySpace/MySpace.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export default function MySpace() {
   return (
     <Style>
       <MSNavbar />
-      <MSProfile />
+      <MSProfile className={mySpace === "profile" ? "hidden" : "hidden"} />
```
Comment (LoicBrassart) [inline frontend/src/pages/MySpace/MySpace.jsx]: ... What ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Useless condition to remove

File: frontend/src/pages/MySpace/MySpace.jsx (hunk 1 above)
Comment (hswiergiel) [reply frontend/src/pages/MySpace/MySpace.jsx]: evil fixed!

**Review by FlorianBartier — APPROVED** [review]
OK

**Review by LoicBrassart — APPROVED** [review]
Better

**Pull request merged**